### DISCUSSION TITLE
British English (or any other really!) translation

### DISCUSSION THREAD

**Discussion by nc011** [openstreetmap/iD#9195] [Q&A]
Hi,

I know iD editor uses American English by default but that we should be able to have a localised British English translation. I'm just wondering how I go about doing this (or for any translation really I guess!)?

Specifically the preset for ``amenity=toilets`` is showing up as "Restroom" even though my browser (Firefox) is in English (GB). I've had a look on Transifex but I don't really know what I'm looking at as it seems like it should be working? 

https://www.transifex.com/openstreetmap/search/?q=source_language%3Aen+source_text%3A%5BToilets%5D+target_language%3Aen_GB

Thanks.

**Comment by 1ec5** [comment]
Yes, there is a British English localization, and it appears to have the correct “Toilets” in Transifex. Unfortunately, the id-tagging-schema package that defines these presets and the Transifex project that translates them are sometimes updated and released on different schedules. My guess is that iD is loading the most up-to-date presets (including openstreetmap/id-tagging-schema#419) but not the British English translation yet.

**Reply by nc011** [reply]
Ahh OK. I see. Thanks for the link. 

> My guess is that iD is loading the most up-to-date presets (including https://github.com/openstreetmap/id-tagging-schema/pull/419) but not the British English translation yet.

I can see @tyrasd added the translation to Transifex on 12 May, which is the same day the change from GB to US wording was committed. So do we just wait for Transifex to release this, or does iD actively need to pull the latest version from there?

**Reply by tyrasd** [reply]
I must have forgot to properly update the translations when doing the last release. I'm currently preparing a fresh release, which should then also fix the missing translation for the Toilets preset. Sorry for the oversight.
